pkg/kapp/diffgraph/change_graph_test.go

```diff
@@ -590,6 +592,41 @@ roleRef:
 	require.Equal(t, expectedOutput, output)
 }
 
+func TestAppCRSvcAccntRBACDelete(t *testing.T) {
```

Comment:
We can also use `_` to have more readable names for tests. 
Let's also add a test for upsert order.

Reply:
Added unit test for upsert.